### ISSUE TITLE
BUG (Mac): Enabling "on top" in examples app hides window in Mission Control

### ISSUE THREAD

**Issue by tonsky** [HumbleUI/JWM#258]
Mission Control meaning what you get when pressing "CTRL+UP" or swiping three fingers up on the touch pad.

_Originally reported at https://github.com/HumbleUI/HumbleUI/issues/61 by @lambdaschmied2_

**Comment by tonsky** [comment]
Seems to be a native macOS behavior https://github.com/electron/electron/issues/6959 that Chrome/Electron still somehow manage to work around https://github.com/electron/electron/blob/d092e6bda4c7ba840f76ef835135f6ac94e02803/shell/browser/native_window_mac.mm#L283-L289

**Comment by bumfo** [comment]
Aren't top windows hidden the intended behavior of mission control? I quite like this behavior because it hides tool windows (which are often set top). I think a new option should be available if a workaround is made, e.g. setHideFromMissionControl. 

A better abstraction would be adding tool & main window separation, and hide all tool windows automatically.

**Comment by mushan0x0** [comment]
> Aren't top windows hidden the intended behavior of mission control? I quite like this behavior because it hides tool windows (which are often set top). I think a new option should be available if a workaround is made, e.g. setHideFromMissionControl.
> 
> A better abstraction would be adding tool & main window separation, and hide all tool windows automatically.

I agree with you, but it doesn't seem to be hiding now, is this a bug? Because I created an alwaysOnTop window with a type of "panel", and I wanted it to be hidden in Mission Control, but it's not hiding in version electron26.
